hardware/OTGWBase.cpp

```diff
@@ -395,19 +410,24 @@ void OTGWBase::ParseLine()
 			SendPressureSensor(0, idx - 1, 255, _status.CH_water_pressure, "CH Water Pressure");
 		}
 
-		_status.Room_temperature = static_cast<float>(atof(results[idx++].c_str()));						SendTempSensor(idx - 1, 255, _status.Room_temperature, "Room Temperature");
-		_status.Boiler_water_temperature = static_cast<float>(atof(results[idx++].c_str()));				SendTempSensor(idx - 1, 255, _status.Boiler_water_temperature, "Boiler Water Temperature");
-		_status.DHW_temperature = static_cast<float>(atof(results[idx++].c_str()));							SendTempSensor(idx - 1, 255, _status.DHW_temperature, "DHW Temperature");
-		_status.Outside_temperature = static_cast<float>(atof(results[idx++].c_str()));						SendTempSensor(idx - 1, 255, _status.Outside_temperature, "Outside Temperature");
-		_status.Return_water_temperature = static_cast<float>(atof(results[idx++].c_str()));				SendTempSensor(idx - 1, 255, _status.Return_water_temperature, "Return Water Temperature");
+		_status.Room_temperature = static_cast<float>(atof(results[idx++].c_str()));
```

Comment:
Could you undo all these new lines ? I don't think any code has changed on this part ?

Reply:
You're right. I will change it back. I accidentally changed it when I was trying to understand the code. The formating looked somewhat strange . The second statement on the last part of the line was not visible in my editor.